fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -439,6 +450,10 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         queueRestoreLocale()
         queueStorageMaintenance()
         queueNimbusFetchInForeground()
+
+        if (settings().enableFxSuggest) {
+            queueFxSuggestIngestion()
```

Comment:
Is this a good way of registering the worker? I modeled it on `queueStorageMaintenance`; our `startPeriodicIngestion` registers its worker in a very similar way to the Places maintenance worker.

Reply:
Forgot to reply back to your comments: Yes, this looks good to me!